Getting error with the following code

panic: Can't run remote command: dial tcp: i/o timeout

``` go
package main

import (
        "fmt"

        "github.com/appleboy/easyssh-proxy"
)

func main() {
        // Create MakeConfig instance with remote username, server address and path to private key.
        ssh := &easyssh.MakeConfig{
                Server: "localhost",
                // Optional key or Password without either we try to contact your agent SOCKET
                //Password: "password",
                Key:     "/.ssh/id_rsa",
                Port:    "22",
                Timeout: 60,
        }

        // Call Run method with command you want to run on remote server.
        stdout, stderr, done, err := ssh.Run("ls -al", 60)
        // Handle errors
        if err != nil {
                panic("Can't run remote command: " + err.Error())
        } else {
                fmt.Println("don is :", done, "stdout is :", stdout, ";   stderr is :", stderr)
        }

}
```

--- a/README.md
+++ b/README.md
@@ -40,6 +40,7 @@ package main
 
 import (
 	"fmt"
+	"time"
 
 	"github.com/appleboy/easyssh-proxy"
 )
@@ -53,7 +54,7 @@ func main() {
 		//Password: "password",
 		Key:     "/.ssh/id_rsa",
 		Port:    "22",
-		Timeout: 60,
+		Timeout: 60 * time.Second,
 	}
 
 	// Call Run method with command you want to run on remote server.

--- a/easyssh_test.go
+++ b/easyssh_test.go
@@ -5,6 +5,7 @@ import (
 	"os/user"
 	"path"
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/assert"
 )
@@ -258,6 +259,7 @@ func TestSCPCommandWithPassword(t *testing.T) {
 		User:     "drone-scp",
 		Port:     "22",
 		Password: "1234",
+		Timeout:  60 * time.Second,
 	}
 
 	err := ssh.Scp("./tests/b.txt", "b.txt")

--- a/example/ssh/ssh.go
+++ b/example/ssh/ssh.go
@@ -2,6 +2,7 @@ package main
 
 import (
 	"fmt"
+	"time"
 
 	"github.com/appleboy/easyssh-proxy"
 )
@@ -15,7 +16,7 @@ func main() {
 		//Password: "password",
 		Key:     "/.ssh/id_rsa",
 		Port:    "22",
-		Timeout: 60,
+		Timeout: 60 * time.Second,
 	}
 
 	// Call Run method with command you want to run on remote server.

